.github/workflows/publish.yml

```diff
@@ -0,0 +1,51 @@
+name: Publish simulator archive
+
+on:
+  release:
+    types: [ created ]
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Build archive
+      id: build_archive
+      run: |
+        mkdir archive
+        NAME="competition-simulator-$(git describe --always --tags).zip"
```

Comment:
You can probably get the tag from `github.ref` much easier, and then perhaps use it instead of the `set-output` stuff below

Reply:
We need the tag basename and `github.ref` gives the full ref name